wrappers/services_gen_test.go

```diff
@@ -1019,3 +1021,158 @@ OOMScoreAdjust=-899
 WantedBy=multi-user.target
 `, mountUnitPrefix, mountUnitPrefix))
 }
+
+func (s *servicesWrapperGenSuite) TestQuotaGroupSlice(c *C) {
```

Comment:
so there was no test for this before?

Reply:
Nope, it seems this was an oversight, so added one while I was down there, and thank you for taking time to get this looked at here on the last day before break!